src/register/coll_reg.cc

```diff
@@ -1,3 +1,5 @@
+/* Copyright © Advanced Micro Devices, Inc., or its affiliates. */
```

Comment:
This file came from NCCL. Either we leave it alone, or we need to add NVIDIA copyrights